ui/src/auction.js

```diff
@@ -0,0 +1,45 @@
+import { E } from '@agoric/eventual-send';
+import { assert, details as X } from '@agoric/assert';
+
+const getCardAuctionDetail = async ({ publicFacet, card }) => {
+  return E(publicFacet).getSessionDetailsForKey(card);
+};
+
+const makeBidOfferForCard = async ({
+  walletP,
+  card,
+  publicFacet,
+  cardPurse,
+  tokenPurse,
+  price,
+}) => {
+  assert(card, X`At least one card must be chosen to purchase`);
+  const invitation = await E(publicFacet).makeBidInvitationForKey(card);
+
+  const cardAmount = {
+    pursePetname: cardPurse.pursePetname,
+    value: harden([card]),
+  };
+  const moneyAmount = {
+    pursePetname: tokenPurse.pursePetname,
+    value: price,
+  };
+
+  const offerConfig = {
+    // JSONable ID for this offer.  This is scoped to the origin.
+    id: Date.now(),
+    invitation,
+    proposalTemplate: {
+      want: {
+        Asset: cardAmount,
+      },
+      give: {
+        Bid: moneyAmount,
+      },
+    },
```

Comment:
Hi @simpletrontdip, sorry, I missed this earlier, but does using the `amounts` here work to make an offer in the wallet?

Reply:
Hi, as in the `complieProposal`, we need the `purseName`, so the original amount may not work
https://github.com/Agoric/agoric-sdk/blob/master/packages/dapp-svelte-wallet/api/src/lib-wallet.js#L849-L879
I think, I tried this before